PayItForward.Data.Services/UsersService.cs

```diff
@@ -36,11 +37,39 @@ public int Count()
             return this.usersRepo.GetAll().Count();
         }
 
-        public UserDTO GetUserById(string id)
+        public UserDTO GetUserById(string userId)
         {
-            var userFromDb = this.usersRepo.GetById(id);
+            var userFromDb = this.usersRepo.GetById(userId);
 
             return this.mapper.Map<UserDTO>(userFromDb);
         }
+
+        public IEnumerable<DonationDTO> GetDonations(string userId)
+        {
```

Comment:
Get Donation can be moved to Donation service

Reply:
It's connected to the story, so I thought that it's better to stay here.

Reply:
I want to get the certain user's donations, so they are connected with the user rather than with any story.